test/test_ovoscope_e2e.py

```
@@ -0,0 +1,220 @@
+from ovos_workshop.intents import IntentBuilder  # noqa: E402
+
+from ovos_adapt.opm import AdaptPipeline  # noqa: E402
+
+PIPELINE_ID = "ovos-adapt-pipeline-plugin"
+CONFIG_KEY = "adapt"
+
+
+class _AdaptHarness(E2EPipelineHarness):
+    PIPELINE_ID = PIPELINE_ID
+    CONFIG_KEY = CONFIG_KEY
+    PLUGIN_CONFIG = {}
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)